pkg/helm/charts/tests/centralservices/testdata/helmtest/scanner-v4.test.yaml

```diff
@@ -231,3 +232,21 @@ tests:
   expect: |
     .configmaps["scanner-v4-indexer-config"].data.["config.yaml"] | fromyaml | .indexer.repository_to_cpe_url | assertThat(contains("https://central.stackrox.svc/api/extensions/scannerdefinitions"))
     .configmaps["scanner-v4-indexer-config"].data.["config.yaml"] | fromyaml | .indexer.name_to_repos_url | assertThat(contains("https://central.stackrox.svc/api/extensions/scannerdefinitions"))
+
+- name: "Scanner V4 is installed by default"
+  set:
+    scannerV4.disable: null
+  expect: |
+    .deployments["scanner-v4-indexer"] | assertThat(. != null)
+    .deployments["scanner-v4-matcher"] | assertThat(. != null)
+    .deployments["scanner-v4-db"] | assertThat(. != null)
+
+- name: "Scanner V4 can be disabled without impacting Scanner V2"
```

Comment:
Testing the escape-hatch for disabling scanner V4 even though this won't be officially communicated as being supported.